examples/jax/cpu-demo/Dockerfile

```diff
@@ -14,6 +14,7 @@ RUN apt-get update && apt-get install -y \
 
 RUN git clone https://github.com/facebookincubator/gloo.git \
     && cd gloo \
+    && git checkout 43b7acbf372cdce14075f3526e39153b7e433b53 \
```

Comment:
Thank you @sandipanpanda!
Should we also install the specific version of `jax` above ?
```
RUN pip install jax absl-py kubernetes
```

Reply:
Sure we can, I was wondering what do you think about keeping it this way to automatically check if Training Operator works with the latest version of JAX from time to time

Reply:
Sure, that is fine for me.
Let's see if the latest version of Jax will work with this version of gloo: 43b7acbf372cdce14075f3526e39153b7e433b53